Expensify/App
# Remove `setTimeout()`. Fix behavior of create menu.

### Details

@Beamanator @luacmartins I don't think we need this `setTimeout()` anymore. Since this was moved into `WelcomeActions` it seems to open the create menu on iOS just fine.

I removed the `setTimeout()` and tested on a few versions of iOS and it works fine.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5296
Regression caused by: https://github.com/Expensify/App/pull/8692

### Tests (iOS)
1. Sign in as a new user
2. Verify the create menu opens


### PR Review Checklist
<details>
<summary><h4>PR Reviewer Checklist</h4></summary>


</details>

### QA Steps
1. Sign in as a new user
2. Verify the create menu opens


### Screenshots


#### Web
https://user-images.githubusercontent.com/32969087/166331089-47caebab-2906-47d4-bb24-6bbe00e81fcf.mp4

#### Mobile Web

https://user-images.githubusercontent.com/32969087/166338511-b1f5286e-8fdb-43ce-b1cd-3753098cf140.mp4


#### Desktop

https://user-images.githubusercontent.com/32969087/166338357-af0692ab-8773-4cb7-8169-d4ff409e89f8.mp4

#### iOS

https://user-images.githubusercontent.com/32969087/165857583-28af9f11-cee4-4ac7-b080-4da6c245f296.mp4

#### Android

https://user-images.githubusercontent.com/32969087/166334381-ca27e0b4-3584-41ce-80ab-ef2ddce4fae1.mp4
